### PR TITLE
PYSVC-201

### PR DESCRIPTION
**Pull request by mikecroft**

### CODE REVIEW

**Comment by mikecroft** [comment]
Preview:
https://mikecroft.gitbooks.io/payara-server-documentation-fork/content/v/PYSVC-201/documentation/extended-documentation/notification-service/notifiers/hipchat-notifier.html

File: documentation/extended-documentation/notification-service/notification-service.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Payara Server version 4.1.1.163 introduced a general Notification service in order to log events which come from other services, such as the [Health Check service](/documentation/extended-documentation/health-check-service/health-check-service.md) or the [Request Tracing service](/documentation/extended-documentation/request-tracing-service/request-tracing-service.md)
 
-The Notification service provides dissemination of notification events through various channels that are being created by Request Tracing Services and/or Health Check Services. As of the version 4.1.1.163, the Notification service integrates with log notification mechanism which can be seen in the `domain.xml`:
+The Notification service provides the ability to disseminate notification events through various channels that are being created by other services, such as the Request Tracing and Health Check Services. The notification service is provided with a number of configurable notifiers, the default being a log notification mechanism which can be seen in this `domain.xml` snippet:
```
Comment (OndrejM) [inline documentation/extended-documentation/notification-service/notification-service.md]: A minor thing - probably a small `S` in `services`:
> ...the Request Tracing and Health Check **services**.

And big `N` in the next sentence:
> The **Notification** service ...

File: documentation/extended-documentation/notification-service/notifiers/notifiers.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Notifiers
+
+Payara Server's notification service ships with a number of notifiers. As of {{ book.currentVersion }}, the available notifiers are as follows:
```
Comment (OndrejM) [inline documentation/extended-documentation/notification-service/notifiers/notifiers.md]: Big `N` to keep consistency:

> Payara Server's **Notification** service

File: documentation/extended-documentation/notification-service/notifiers/hipchat-notifier.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+# HipChat Notifier
+HipChat is a team chat application created by Atlassian. It can be used for teams to discuss topics in separate "rooms", but also supports integrations from services to notify users of events.
+
+Payara Server has now included functionality to send notifications from the [notification service](/documentation/extended-documentation/notification-service/notification-service.md) to a HipChat room
```
Comment (OndrejM) [inline documentation/extended-documentation/notification-service/notifiers/hipchat-notifier.md]: Big `N` to keep consistency:

> ... from the [**Notification** service]...

File: documentation/extended-documentation/notification-service/notifiers/hipchat-notifier.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+
+## Requirements
+To use HipChat as a notification target, you must have a HipChat account already created and a room where notifications will be sent.
```
Comment (OndrejM) [inline documentation/extended-documentation/notification-service/notifiers/hipchat-notifier.md]: Maybe add a comma between `created` and `add a room`?

> account already created, and a room

Comment (OndrejM) [inline documentation/extended-documentation/notification-service/notifiers/hipchat-notifier.md]: The following wording sounds better to me:

> ... you must have an already created HipChat account

Or just `you must have a HipChat account`

**Comment by mikecroft** [comment]
RE: the "assets" folder thing, the gitbook editor uploader does that automatically. I did move it out of assets into images, but when working with lots of images, it made more sense to me to just go with the default. I didn't think anyone else was using the top level images directory, so my intention was to move them all into `assets` after

**Comment by mikecroft** [comment]
Should all have been addressed!

**Pull request merged**